Library/Formula/libbluray.rb

```diff
@@ -1,9 +1,7 @@
-require "formula"
-
 class Libbluray < Formula
   homepage "https://www.videolan.org/developers/libbluray.html"
-  url "ftp://ftp.videolan.org/pub/videolan/libbluray/0.6.2/libbluray-0.6.2.tar.bz2"
-  sha1 "a1ab8c8c9310680fb1fe6a58f9fc5430613600fe"
+  url "ftp://ftp.videolan.org/pub/videolan/libbluray/0.7.0/libbluray-0.7.0.tar.bz2"
+  sha1 "12baf71accac2ae9efe7fb077999821ea1430d7b"
```

Comment:
> So would anyone object to me using SHA-256 by default from now on for everything except projects that only supply a SHA-1?

I would, yes. Git relies on SHA-1 for security and we rely on that. When you've convinced Git to move to `sha256` we'll do the same :wink: